e2e-tests/mdx/src/pages/frontmatter-graphql-query.js

```diff
@@ -8,23 +8,16 @@ export default function PageRunningGraphqlResolversOnJSFrontmatterTestInputs({
 }
 
 export const query = graphql`
-  {
-    allMdx(filter: { slug: { glob: "frontmatter-engine/*" } }) {
-      nodes {
-        frontmatter {
-          title
-        }
-        body
-        excerpt
-        tableOfContents
-        timeToRead
-        wordCount {
-          paragraphs
-          sentences
-          words
-        }
-        mdxAST
-      }
-    }
-  }
-`
+         {
```

Comment:
Indentation is off here

Reply:
This is what prettier wanted 🤔

Reply:
Nice, it formats it back and forth in my editor :D

Forced it now to be aligned left!